.eslintrc.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
     mocha: true,
     browser: true,
   },
+  parser: "babel-eslint",
```

Comment:
is this needed for dynamic import?

Reply:
No, it's needed because the current `eslint` doesn't think `import()` is allowed—it says it's a parse error. It expects only top-level import:

```js
import 'foo';
```

Reply:
Sorry if I wasn't clear, that's what I meant. The linter isn't recognizing dynamic import.